### PR TITLE
Stoivo/inline diff big files

### PR DESCRIPTION
**Pull request by kaste**
@stoivo I cannot add commits to PR's. So this one works for me.

### CODE REVIEW

File: core/commands/inline_diff.py [hunk 1]
```diff
@@ -343,8 +340,7 @@ class GsInlineDiffFocusEventListener(EventListener):
     latest file status when the view regains focus.
     """
 
-    def on_activated(self, view):
-
+    def on_activated_async(self, view):
         if view.settings().get("git_savvy.inline_diff_view") is True:
             view.run_command("gs_inline_diff_refresh")
```
Comment (kaste) [inline core/commands/inline_diff.py]: That's a mistake. If you run a command like `view.run_command("gs_inline_diff_refresh")` it always starts at the main thread.

Look at how it is done for the diff view 

https://github.com/divmain/GitSavvy/blob/29642a3f5731be4cbd63f9acd56b4b714fbc0001/core/commands/diff.py#L129-L138

`sync` being the default you call it with `sync=False` in on_activated and in the main gs_inline_diff command.

Comment (stoivo) [reply core/commands/inline_diff.py]: Do you know if there is a reliable way to add sleep to be blocking so I can check it works like I expect

Comment (kaste) [reply core/commands/inline_diff.py]: Don't know what you're trying to do. `time.sleep` is always blocking. It is also important to check on which thread you run. (E.g. `import threading; print threading.current_thread()`)

Comment (stoivo) [reply core/commands/inline_diff.py]: Take a look now

**Comment by kaste** [comment]
LGTM! But I had to fix the commit message bc you got it backwards.

**Comment by stoivo** [comment]
Good, it was late last night. thanks

**Pull request merged**